src/main/java/dinkplugin/domain/ChatNotificationType.java

```diff
@@ -0,0 +1,44 @@
+package dinkplugin.domain;
+
+import net.runelite.api.ChatMessageType;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static net.runelite.api.ChatMessageType.*;
+
+/**
+ * A condensed version of {@link ChatMessageType} for more user-friendly configuration.
+ */
+public enum ChatNotificationType {
+    GAME("Game Engine", GAMEMESSAGE, ENGINE, MESBOX, LOGINLOGOUTNOTIFICATION, PLAYERRELATED),
+    CLAN("Clan Notifications", CLAN_MESSAGE, CLAN_GIM_MESSAGE, CLAN_GUEST_MESSAGE, FRIENDSCHATNOTIFICATION),
+    TRADES("Trades and Duels", TRADE, TRADE_SENT, TRADEREQ, CHALREQ_TRADE, CHALREQ_FRIENDSCHAT, CHALREQ_CLANCHAT),
+    CHAT("User Messages (Discouraged)", PUBLICCHAT, PRIVATECHAT, PRIVATECHATOUT, MODCHAT, MODPRIVATECHAT, FRIENDSCHAT, CLAN_CHAT, CLAN_GUEST_CHAT, AUTOTYPER, MODAUTOTYPER);
```

Comment:
Why do we discourage user message notifications?

Reply:
Since patterns are not type-specific, a troll could send a chat message that looks like a game message the user cares about, triggering a "false" notif